AddInstallationScriptTest.cmake

```diff
@@ -0,0 +1,25 @@
+macro(add_installation_script_test name path)
+
+  # Copy the source to the binary tree 
+  configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/${path}/${name}
+    ${CMAKE_CURRENT_BINARY_DIR}/${path}/${name}
+    COPYONLY
+  )
+  configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/${path}/${name}-usage
+    ${CMAKE_CURRENT_BINARY_DIR}/${path}/${name}-usage
```

Comment:
If you pass in `name` with suffix `.sh` then usage is going to be `${name}.sh-usage` ...